openedx/core/djangoapps/config_model_utils/admin.py

```diff
@@ -28,9 +29,48 @@ class StackedConfigModelAdmin(ConfigurationModelAdmin):
     """
     form = StackedConfigModelAdminForm
 
-    def get_fields(self, request, obj=None):
+    raw_id_fields = ('course',)
+
+    def get_fieldsets(self, request, obj=None):
+        return (
+            ('Context', {
+                'fields': self.key_fields,
+                'description': _(
+                    'These define the context to enable this configuration on. '
+                    'If no values are set, then the configuration applies globally. '
+                    'If a single value is set, then the configuration applies to all courses '
+                    'within that context. At most one value can be set at a time.<br>'
```

Comment:
For internationalizing strings is it okay to have HTML in it? Or is that something that should be interpolated?

Reply:
Fixed